addon/README.md

```diff
@@ -1,5 +1,5 @@
 # Test Pilot
-The addon where ideas come to idea
+The add-on where ideas come to idea
```

Comment:
:+1: to the comedic historical tagline. But, not sure if we want to  update it to be more descriptive. Stealing from the package.json `description`:

> _Test Pilot is a privacy sensitive user research program focused on getting new features into Firefox faster._